cli/src/semgrep/output.py

```diff
@@ -245,7 +245,7 @@ def _handle_semgrep_error(self, error: SemgrepError) -> None:
             if self.settings.output_format == OutputFormat.TEXT and (
                 error.level != Level.WARN or self.settings.verbose_errors
             ):
-                logger.error(with_color(Colors.red, str(error)))
```

Comment:
each exception has complex formatting code, but then this thing just recolored everything to red :D